themes/classic/_dev/css/partials/_commons.scss

```diff
@@ -454,3 +454,13 @@ input, textarea {
   margin: 0;
   border-color: rgba(0, 0, 0, 0.25);
 }
+// modal-dialog-centered class added from bootstrap 4.0.0 adapted for bootstrap 4.0.0-alpha.5
+.modal-dialog-centered {
+  display: -webkit-box;
```

Comment:
Missing `display: -moz-box;` &   `display: -webkit-flex;`, same for `align-items`.
Maybe you can use some mixins: https://www.npmjs.com/package/flexbox-polyfills :)

Reply:
it's just to prevent a copy paste of many properties every time we want to use `display: flex` :)

Reply:
There's no need for prefixes, they are added automatically by postcss